src/gistEncoder.js

```diff
@@ -0,0 +1,51 @@
+const fs = require("fs").promises;
+const path = require("path");
+const { OpusEncoder } = require("@discordjs/opus");
+const WaveFile = require("wavefile").WaveFile;
+
+async function encodePCM(file) {
+  const wav_file = await fs.readFile(file);
+  const wav = new WaveFile(wav_file);
+
+  // Check some of the file properties
+  console.log("Sample rate:::\t\t", wav.fmt.sampleRate);
+  console.log("Nr channels:::\t\t", wav.fmt.numChannels);
+
+  const encoder = new OpusEncoder(wav.fmt.sampleRate, wav.fmt.numChannels);
+  // encoder.setBitrate(48000); // make sure it sounds like ****
```

Comment:
This line makes no difference in the output, sample rate is always low. Wonder why 🤔

Reply:
sample rate and bitrate is not the same thing.

Reply:
@xnorpx true, bit rate `!==` bit depth `!==` sample rate. I guess my confusion here is that changing the value didn't made any difference. What would be a value that would be audible better quality here?

Reply:
set it to 6000 and then you can hear difference. the quality saturates around 36kbit/s for speech depending on wideband or not. higher bitrates you can probably only tell difference in music.